.lintstagedrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "*.{css,less,scss,graphql,gql,html,js,jsx,json,md,markdown,mdown,mkdn,mdx,ts,tsx,vue,yaml,yml}": "prettier --write",
```

Comment:
I always thought that maybe we should improve it (just idea):
```
"*.*": "prettier --write --ignore-unknown"
```

Reply:
`--ignore-unknown` should easy to add.

Reply:
@thorn0 What do you think?

Reply:
Please open a new issue for it.